.jenkins/pipelines/code-coverage.Jenkinsfile

```diff
@@ -5,79 +5,325 @@
+                checkout([$class: 'GitSCM',
+                    branches: [[name: "pr/${PULL_REQUEST_ID}"]],
+                    extensions: [],
+                    userRemoteConfigs: [[
+                        url: 'https://github.com/deislabs/mystikos',
+                        refspec: "+refs/pull/${PULL_REQUEST_ID}/merge:refs/remotes/origin/pr/${PULL_REQUEST_ID}"
+                    ]]
+                ])
+            }
+        }
+        stage('Verify commit sync') {
+            when {
+                expression { params.COMMIT_SYNC != "" }
+            }
+            steps {
+                // Check if the checked out commit is the same across all parallel builds
+                script {
+                    def GIT_COMMIT_ID = sh(
+                        returnStdout: true,
+                        script: "git log --max-count=1 --pretty=format:'%H'"
+                    ).trim()
+                    if ( GIT_COMMIT_ID != params.COMMIT_SYNC ) {
+                        error("Checked out commit (${GIT_COMMIT_ID}) does not match commit from upstream job (${params.COMMIT_SYNC})")
+                    } else {
+                        println("Checked out commit (${GIT_COMMIT_ID}) matches upstream job (${params.COMMIT_SYNC}). Continuing with build.")
                     }
-                    axis {
-                        name 'TEST_PIPELINE'
-                        values 'Unit', 'Solutions', 'DotNet', 'Azure-SDK'
+                }
+            }
+        }
+        stage('Init Config') {
+            steps {
+                sh """
+                   # Initialize dependencies repo
+                   ${JENKINS_SCRIPTS}/global/wait-dpkg.sh
+                   ${JENKINS_SCRIPTS}/global/init-config.sh
+
+                   # Install global dependencies
+                   ${JENKINS_SCRIPTS}/global/wait-dpkg.sh
+                   ${JENKINS_SCRIPTS}/global/init-install.sh
+                   """
+            }
+        }
+        stage('Setup AZ CLI') {
+            steps {
+                withCredentials([string(credentialsId: 'Jenkins-ServicePrincipal-ID', variable: 'SERVICE_PRINCIPAL_ID'),
+                                    string(credentialsId: 'Jenkins-ServicePrincipal-Password', variable: 'SERVICE_PRINCIPAL_PASSWORD'),
+                                    string(credentialsId: 'ACC-Prod-Tenant-ID', variable: 'TENANT_ID'),
+                                    string(credentialsId: 'ACC-Prod-Subscription-ID', variable: 'AZURE_SUBSCRIPTION_ID')]) {
+                    sh """
+                        ${JENKINS_SCRIPTS}/azure-sdk/install-azure-cli.sh
+                        ${JENKINS_SCRIPTS}/azure-sdk/login-azure-cli.sh
+                    """
+                }
+            }
+        }
+        stage('Build repo source') {
+            steps {
+                sh """
+                   ${JENKINS_SCRIPTS}/global/make-world.sh
+                   """
+            }
+        }
+        stage("Install package") {
```

Comment:
There are two conflicting stages here -- building Mystikos from source and then installing it from deb/tar. Which one should be done?

Reply:
As we discussed, I removed the package stage